circle.yml

```diff
@@ -0,0 +1,55 @@
+machine:
+  environment:
+    ARCH: "i686"
+    BUILDOPTS: "-j3 VERBOSE=1 FORCE_ASSERTIONS=1 LLVM_ASSERTIONS=1"
+    TESTSTORUN: "all"
+    JULIA_CPU_CORES: 2
```

Comment:
the free plan only gives you one build worker to start right? it just has 2 cores inside that one job